client.go

```diff
@@ -556,6 +555,28 @@ func (c *Client) GetBiosConfiguration(ctx context.Context) (biosConfig map[strin
 	return biosConfig, err
 }
 
+func (c *Client) SetBiosConfiguration(ctx context.Context, biosConfig map[string]string) (err error) {
```

Comment:
Do you think it would be possible to have the map keys of `biosConfig` be some `const` value? Or maybe use a structure with fields for specific settings? Using a map with string keys and values makes it hard for the caller to know what settings are available, or supported.